src/style/shared/button.less

```diff
@@ -259,38 +262,8 @@ button {
             opacity: 1;
             transform: rotate(0);
         }
-    }
-}
-
-// Fixing for Retina
-@media (-webkit-min-device-pixel-ratio: 2) {
```

Comment:
Just double checking in case this was a merge issue or something ...  What changed such that this retina fix is no longer necessary?